spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+RSpec.describe 'post#how', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'Habtamu', photo: 'My-photo', bio: 'Software Engineer')
+
+    @user2 = User.create(name: 'Solomon', bio: 'Elec Engineer',
+                         photo: 'sol-pic')
+
+    @post = Post.create(
+      title: 'Blog App dev',
+      text: 'Class association in detail there',
+      author_id: @user.id
+    )
+
+    Comment.create(text: 'Blog in rails is challenging', author_id: @user2.id, post_id: @post.id)
+    Comment.create(text: 'Yet you will adapt to it Sol', author_id: @user.id, post_id: @post.id)
+    Comment.create(text: 'Thanks', author_id: @user2.id, post_id: @post.id)
+    Like.create(post_id: @post.id, author_id: @user2.id)
+  end
+
+  describe 'post#show test group' do
+    it 'displays the post title' do
+      visit user_posts_path(@user, @post.id)
+      expect(page).to have_content('Class association in detail there')
+    end
+
+    it 'displays the post author' do
+      visit user_posts_path(@user, @post.id)
+      expect(page).to have_content('Amare')
+    end
+
+    it 'displays the comments counter' do
+      visit user_posts_path(@user, @post.id)
+      expect(page).to have_content('Comments: 3')
+    end
```

Comment:
- [ ] Kindly make sure to implement one left featured tests for the Post show page: 👇🏻

![Screenshot](https://github.com/Habtamu210/Blog-App/assets/79058364/a62dfae6-40b6-47cb-8002-adaf8692aa76)